packages/neos-ui/src/Containers/LeftSideBar/NodeTreeSearchBar/index.js

```diff
@@ -72,19 +79,18 @@ class NodeTreeSearchBar extends PureComponent {
     }
 
     handleSearchToggle = () => {
-        this.setState({
-            searchToggled: !this.state.searchToggled
-        });
+        const {toggleSearchBar} = this.props;
+        toggleSearchBar();
     }
 
     render() {
-        const {i18nRegistry} = this.props;
-        const {searchToggled, searchValue, searchFocused, filterNodeType} = this.state;
+        const {i18nRegistry, isHiddenSearchBar} = this.props;
+        const {searchValue, searchFocused, filterNodeType} = this.state;
         const searchLabel = i18nRegistry.translate('search', 'Search', {}, 'Neos.Neos', 'Main');
 
         const searchToggleClassName = mergeClassNames({
             [style.searchToggleButton]: true,
-            [style['searchToggleButton--active']]: searchToggled
+            [style['searchToggleButton--active']]: !isHiddenSearchBar
```

Comment:
Why did you reverse it? I like the old name better :)

Reply:
Can also name it like this.